src/test/java/uk/ac/ebi/ampt2d/metadata/MetadataApplicationTest.java

```diff
@@ -173,11 +175,12 @@ private String postTestStudy(String name) throws Exception {
     public void postAnalysis() throws Exception {
         String assemblyUrl = postTestAssembly("GRCh37", "p2",
                 Arrays.asList("GCA_000001405.3", "GCF_000001405.14"));
-        String studyUrl = postTestStudy("test human study");
+        String studyUrl = postTestStudy("EGAS0001", 1, "test_human_study");
 
         MvcResult mvcResult = mockMvc.perform(post("/analyses")
                 .content("{ " +
-                        "\"name\": \"test human analysis\"," +
+                        "\"id\":{ \"accession\": \"EGAA0001\",\"version\": " + 1 + "}," +
```

Comment:
If "1" is not coming from a constant, write it inside the string to avoid unnecessary concatenations.